paymentsheet/src/test/java/com/stripe/android/paymentelement/confirmation/DefaultConfirmationHandlerTest.kt

```diff
@@ -28,12 +29,16 @@ import kotlinx.coroutines.test.UnconfinedTestDispatcher
 import kotlinx.coroutines.test.runTest
 import kotlinx.coroutines.withTimeout
 import kotlinx.parcelize.Parcelize
+import org.junit.Rule
 import org.mockito.kotlin.mock
 import kotlin.test.Test
 import kotlin.time.Duration.Companion.minutes
 import kotlin.time.Duration.Companion.seconds
 
 class DefaultConfirmationHandlerTest {
+    @get:Rule
+    val coroutineTestRule = CoroutineTestRule()
```

Comment:
Using `Dispatcher.Main.immediate` directly. Could pass it in instead of using this rule, but in practice it doesn't feel much different.